ember-electron/test-main.js

```diff
@@ -14,23 +15,29 @@ let [, , indexUrl] = process.argv;
 indexUrl = indexUrl.replace(/__amp__/g, '&');
 console.log(`URL: ${indexUrl}`);
 let {
-  pathname: indexPath,
+  pathname: originalIndexPath,
   search: indexQuery,
 } = url.parse(indexUrl);
 // When we extract the pathname from an absolute path on windows, it starts
 // with '/C:/', and the leading slash confuses everything, so we need to strip
 // it.
 if (process.platform === 'win32') {
-  indexPath = indexPath.slice(1);
+  originalIndexPath = originalIndexPath.slice(1);
 }
-indexPath = resolve(indexPath);
+
+// Copy index-electron.html to a location not managed by broccoli so that it
+// doesn't get deleted when a change is detected.
+originalIndexPath = resolve(originalIndexPath);
+let emberAppDir = resolve(dirname(originalIndexPath), '..');
+let indexPath = resolve(emberAppDir, '../..', basename(originalIndexPath));
```

Comment:
I think the best way would be to just create a new temp directory using the `tmp` module. The `tmp` module automatically deletes its directories when the process exits.